Formula/inlets.rb

```diff
@@ -0,0 +1,126 @@
+class Inlets < Formula
+  desc "Expose your local endpoints to the Internet"
+  homepage "https://github.com/alexellis/inlets"
+  url "https://github.com/alexellis/inlets.git",
+      :tag      => "0.6.2",
+      :revision => "fbc9f4fe98cf48e44300736140bd013d64d6f980"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["XC_ARCH"] = "amd64"
```

Comment:
Is this required for the formula to work?

Reply:
If you mean setting the Arch, then I don't believe so. It should pick that up from the build env

Reply:
Could you remove it then?